### ISSUE TITLE
Network check in upload script should be done after the platform check

### ISSUE THREAD

**Issue by dreese** [Instabug/Instabug-iOS#15]
Checking the internet connection can take up to 10 seconds, and if we are going to skip the upload anyway (because it's a simulator build), then there is no reason to have the delay.

Just need to move this code:
```
#Check internet connection
if [ ! "`ping -c 1 instabug.com`" ]; then
  exit 0
fi
```

To be after this code, instead of before it:
```
# Check for simulator builds
if [ "$EFFECTIVE_PLATFORM_NAME" == "-iphonesimulator" ]; then
  if [ "${SKIP_SIMULATOR_BUILDS}" ] && [ "${SKIP_SIMULATOR_BUILDS}" -eq 1 ]; then
    echo "Instabug: Skipping simulator build"
    exit 0
  fi
fi
```

**Comment by msoliman** [comment]
That makes sense, I'll make sure to fix this in our next release.

**Comment by dreese** [comment]
Thanks!

**Comment by msoliman** [comment]
It's now fixed in 0e50224263e81c6363d1787d5f2ea8359863fdf1
